loopout.go

```diff
@@ -1320,6 +1320,13 @@ func (s *loopOutSwap) sweep(ctx context.Context,
 	return nil
 }
 
+func (s *loopOutSwap) swapInfo() *SwapInfo {
+	swapInfo := s.swapKit.swapInfo()
+	swapInfo.OutgoingChanSet = s.OutgoingChanSet
```

Comment:
Here the `OutgoungChanSet` is passed by reference, which may be dangerous as ownership is shared.

Reply:
Yes exactly. By sharing ownership I meant that the owner of `OutgoingChanSet` is the `loopOutSwap` object. By passing it as reference without first copying it we also share the ownership if that slice with the newly created `SwapInfo` object.